src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -21,30 +21,33 @@
 import reposense.git.exception.GitCloneException;
 import reposense.parser.ArgsParser;
 import reposense.parser.AuthorConfigCsvParser;
-import reposense.parser.CsvParserTest;
 import reposense.parser.GroupConfigCsvParser;
+import reposense.parser.GroupConfigParserTest;
 import reposense.parser.InvalidLocationException;
 import reposense.parser.ParseException;
 import reposense.parser.RepoConfigCsvParser;
+import reposense.parser.RepoConfigParserTest;
 import reposense.report.ReportGenerator;
 import reposense.util.FileUtil;
 import reposense.util.InputBuilder;
 import reposense.util.TestUtil;
 
 public class RepoConfigurationTest {
-    private static final Path IGNORE_STANDALONE_TEST_CONFIG_FILES = new File(CsvParserTest.class.getClassLoader()
+    private static final Path IGNORE_STANDALONE_TEST_CONFIG_FILES = new File(RepoConfigParserTest.class.getClassLoader()
```

Comment:
I'm not really sure why the existing code has a reference to `CsvParserTest` here.
The resource loading works even if we use `RepoConfigurationTest.class.getClassLoader()...`
@fzdy1914 @jylee-git, do you know why `CsvParserTest` is used here?

Reply:
Agrees to what you say. The test is written in the early state of this project. Maybe we can consider to just use `RepoConfigurationTest.class.getClassLoader()` here.

Reply:
@andlimey Can you make the changes mentioned by @anubh-v ?